vars/stageDeployToOpenshift.txt

```diff
@@ -0,0 +1,19 @@
+stageDeployToOpenshift deploys the service.
+
+A deployment is more than just updating an image, it might also involve adding
+an environment variable, or changing constraints etc. Because of this, a
+deployment should be one atomic JSON patchset.
```

Comment:
This sounds a bit like that we would want to have someone author a JSON patchset which is then applied.

Reply:
No, should be automatic - need to reword.